README.md

```diff
@@ -1,37 +1,135 @@
-# TYMA-2021
+# TyMA-2020
 
-Code, data and programs for TYMA exercises.
+## Index
 
-## Use
+- [1. Downloading this repository](#1.-Downloading-this-repository)
+- [2. Installing C compiler, Python and dependencies](#2.-Installing-C-compiler,-Python-and-dependencies)
+- [3. C compiler](#3.-C-compiler)
+- [4. Installing Python 3.x](#4.-Installing-Python-3.x)
+- [5. Python packages](#5.-Python-packages)
+- [6. Compiling C code](#6.-Compiling-C-code)
+- [7. Running the generators](#7.-Running-the-generators)
+  - [7.1 generate_dotplot](#7.1-generate_dotplot)
+  - [7.2 generate_dotplot_word_thresh](#7.2-generate_dotplot_word_thresh)
+- [8. Generating images with Python](#8.-Generating-images-with-Python)
 
-1. First clone or download this repository from [here](https://github.com/estebanpw/TYMA-2021)
+## 1. Downloading this repository
 
-2. If on linux, run `make all` to compile. If on windows, execute the "make-windows.bat" 
+To save this git repository to your machine, just execute
 
-3. Make sure you have python 3 installed. You can download it from here: https://www.python.org/downloads/
+```bash
+git clone https://github.com/olegbrz/TYMA-2021
+```
 
-4. Add the following libraries to your python installation:
- - pip install numpy
- - pip install matplotlib
- 
-## Running dotplots
+If you don't want to install Git (although you should...), you can just save it as `.zip`.
 
-Run the dotplot generator as `./bin/generate_dotplot fastaX fastaY outdotplot`
+## 2. Installing C compiler, Python and dependencies
 
-This will generate a dotplot file which you can plot using `python ./bin/dotplot.py outdotplot`
+In order to use this program, you need to install:
 
-And a `.png` file will be created on the folder
+- C compiler
+- Python 3.x
+- numpy and matplotlib Python packages
 
-## Creating word dictionaries
+## 3. C compiler
 
-Run `bin/dictionary fastas/hemoglobine.fasta output/dictionary`
+MinGW compiler is recommended. It can be downloaded from:
```

Comment:
Can you make this section to distingush between Windows and Linux? (e.g. indicate that MinGW is recommended for Windows whereas for linux you most probably have already the gcc compiler installed and on the path)